sentences/nl/light_HassLightSet.yaml

```diff
@@ -1,4 +1,23 @@
 language: nl
 intents:
   HassLightSet:
-    data: []
+    data:
+      # brightness
+      - sentences:
+          - "zet <name> op <brigtness> procent"
+          - "zet [de] helderheid van <name> op <brightness> [procent]"
+          - "verander [de] helderheid van <name> naar <brightness> [procent]"
+          - "zet [de] felheid van <name> op <brightness> [procent]"
+          - "verander [de] felheid van <name> naar <brightness> [procent]"
+          - "zet [de] intensiteit van <name> op <brightness> [procent]"
+          - "verander [de] intensiteit van <name> naar <brightness> [procent]"
+      - sentences:
+          - "zet [de] helderheid in [de][het] <area> op <brightness> [procent]"
+          - "zet <area> helderheid op <brightness> [procent]"
+        slots:
+          name: "all"
+      # color
+      - sentences:
+          - "zet <name> op {color}"
+          - "verander [[de] kleur van] <name> naar {color}"
+          - "maak <name> {color}"
```

Comment:
Colors also need to be defined in the common file. I would limit this PR to just brightness, do color once it's merged.